src/components/Editor/Organisations/Organisations.vue

```diff
@@ -27,14 +27,18 @@
                     <v-card
                       v-if="link.organisation.name"
                       key="view"
-                      class="flexCard grey lighten-3"
+                      class="flexCard lighten-3"
+                      :class="{'grey': !link.isLead, 'green': link.isLead}"
```

Comment:
please change this to (merge two class in one bound class like below)

:class=['flexCard lighten-3',{'grey': !link.isLead, 'green': link.isLead}]